Pipfile

```diff
@@ -0,0 +1,12 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+"e1839a8" = {editable = true, path = ".", extras = ["optional", "dev", "tests"]}
```

Comment:
Isn't this problematic?

Reply:
No. https://github.com/pypa/pipfile/issues/93

Reply:
It is cryptic-looking though.

Reply:
https://github.com/pypa/pipfile/issues/93#issuecomment-375068485 Can we change the name to just 'sentry'?